substrate/client/authority-discovery/src/worker.rs

```diff
@@ -581,7 +597,111 @@ where
 
 				debug!(target: LOG_TARGET, "Failed to put hash '{:?}' on Dht.", hash)
 			},
+			DhtEvent::PutRecordRequest(record_key, record_value, publisher, expires) => {
+				if let Err(e) = self
+					.handle_put_record_requested(record_key, record_value, publisher, expires)
+					.await
+				{
+					debug!(target: LOG_TARGET, "Failed to handle put record request: {}", e)
```

Comment:
nit; this is unlikely to happen, should we bump this up to a warn for visibility? (similar to litep2p)

Reply:
It is consistent with how we treat processing errors in this file, so I think debug is the right level here.